src/Components/Publishing/Display/DisplayAd.tsx

```diff
@@ -49,8 +70,19 @@ export const DisplayAd: SFC<DisplayAdProps> = props => {
     return null
   }
 
+  if (useDidMount) {
```

Comment:
This looks like incorrect usage of `useDidMount`. I'd expect this statement to always return true, because `useDidMount` is a function.

Looking at the code in `useDidMount`, I'd expect the usage to look like this: 

```js
const didMount = useDidMount()

...

if (didMount) {
  ...
}
```

I also think the initial call to `useDidMount()` should be moved up to line 46, to sit with the other hooks calls. One critical thing about hooks is that they are called in the same order every single time. Right now, that's going to happen in this function even if we call `useDidMount()` on line 73...but grouping the hooks calls at the top of the function prevents us from inserting an early-return in between them in the future. (Which would cause a conditional hooks call, which would not work.)